File: backend/src/Designer/Models/DeploymentResponse.cs
using System.Collections.Generic;
using Altinn.Studio.Designer.Repository.Models;
using Altinn.Studio.Designer.Services.Models;

namespace Altinn.Studio.Designer.Models
{
    /// <summary>
    /// Deployment
    /// </summary>
    public class DeploymentResponse
    {
        public List<DeploymentEntity> PipelineDeploymentList { get; set; }

        public List<KubernetesDeployment> KubernetesDeploymentList { get; set; }
    }
}

File: frontend/packages/shared/src/types/api/DeploymentResponse.ts
import type { KubernetesDeployment } from './KubernetesDeployment';
import type { PipelineDeployment } from './PipelineDeployment';

export interface DeploymentResponse {
  pipelineDeploymentList: PipelineDeployment[];
  kubernetesDeploymentList: KubernetesDeployment[];
}
